customOverlay.wake

```diff
@@ -34,13 +38,17 @@ global def makeDevicetreeChosenMemoryRam nodePath regTuple regOffset =
 global def makeDevicetreeChosenMemoryItim nodePath regTuple regOffset =
   DevicetreeChosenMemory "metal,itim" nodePath regTuple regOffset
 
+global def makeDevicetreeChosenBootHart nodePath =
+  DevicetreeChosenHart "metal,boothart" nodePath
+
 tuple DevicetreeChosenNode =
   global Entry: DevicetreeChosenMemory
   global Ram:   DevicetreeChosenMemory
   global Itim:  Option DevicetreeChosenMemory
+  global Hart:  Option DevicetreeChosenHart
 
-global def makeDevicetreeChosenNode entry ram itim =
-  DevicetreeChosenNode entry ram itim
+global def makeDevicetreeChosenNode entry ram itim hart =
```

Comment:
After thinking about it, you're right, we don't want to add a `hart` argument to this function because we don't want to force everyone to specify the boot hart and break this API.

Instead, the makeDevicetreeChosenNode should follow the default selection criteria, and the user should be able to override the choice by calling `setDevicetreeChosenNodeHart` on the result of `makeDevicetreeChosenNode`.